opentelekomcloud/services/cce/resource_opentelekomcloud_cce_cluster_v3.go

```diff
@@ -197,9 +197,13 @@ func ResourceCCEClusterV3() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
-			"kube_proxy_mode": { // can't be set via API currently
+			"kube_proxy_mode": {
 				Type:     schema.TypeString,
+				ForceNew: true,
+				Optional: true,
 				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"ipvs", "iptables"}, true),
```

Comment:
It's better to be split in lines like 
```go
validation.StringInSlice(
	[]string{"ipvs", "iptables"}, true,
),

Reply:
@outcatcher it gets auto-formatted to what you see above when you try to do that :)

Reply:
@artem-lifshits 
There are examples of working option:
```go
				ValidateFunc: validation.StringInSlice([]string{
					"mongos", "shard", "config", "replica",
				}, true),
```

Reply:
Ah I see, so the part after the slice would be put on the next line?

Reply:
The slice opening bracket on previous live and closing - on the next 🙃